spec.bs

```diff
@@ -419,9 +420,15 @@ The <dfn for=Navigator method>runAdAuction(|config|)</dfn> method steps are:
     1. [=Queue a global task=] on [=DOM manipulation task source=], given |global|, to resolve |p|
       with null.
   1. Otherwise:
-    1. [=Queue a global task=] on the [=DOM manipulation task source=], given |global|, to resolve
-      |p| with |winner|'s [=generated bid/ad descriptor=]. TODO: resolve |p| with urn-uuid, instead
-      of a URL.
+    1. Let |urn| and |fencedFrameConfig| be the result of TODO: constructing the relevant
+       {{FencedFrameConfig}} with |winner|'s [=generated bid/ad descriptor=].
+    1. [=Upon fulfillment=] of |auctionConfig|'s [=auction config/resolve to config=] with value

```

Comment:
Upon fulfillment must happen on the main thread so we have to pull this out to before we go in parallel. It can mutate primitive that the in parallel context can _wait_ on, but that's about it. See for example https://github.com/WICG/turtledove/pull/585.

Reply:
I changed it to match the other PR. Instead of incrementing/decrementing a counter, I just used whether the value was a boolean (i.e. whether it had been resolved)... Not sure if that's ok.